presidio-anonymizer/presidio_anonymizer/entities/anonymizer_request.py

```diff
@@ -106,12 +107,12 @@ def __get_anonymizer(self, transformation):
         Extract the anonymizer class from the anonymizers list.
 
         :param transformation: a single transformation value
-        :return: anonymizer_class
+        :return: anonymizer
```

Comment:
if you've dropped the reference for it being a class, add it as type hint

Reply:
Done.